shop/src/main/java/ro/msg/learning/shop/dto/OrderInputDto.java

```diff
@@ -0,0 +1,23 @@
+package ro.msg.learning.shop.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ro.msg.learning.shop.domain.Customer;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class OrderInputDto {
+    private UUID id;
+    private Customer customer;
```

Comment:
there should be no use of an entity in a dto.

Reply:
done, removed this dto